examples/promise-co-await/.babelrc

```diff
@@ -0,0 +1,4 @@
+{
+  "presets": ["es2015", "stage-2"],
```

Comment:
probably at least node with native generators support, because promise api is optional and lazy-loaded.
This `.babelrc` is just to allow to run examples easily ( run npm install in examples folder + propably bablel-node for those  requiring transpilation )